awscli/customizations/ec2decryptpassword.py

```diff
@@ -79,11 +82,11 @@ def add_to_params(self, parameters, value):
             path = os.path.expanduser(path)
             if os.path.isfile(path):
                 self._key_path = path
-                service_name = self._operation.service.endpoint_prefix
+                service_name = \
+                    self._operation_model.service_model.service_name
```

Comment:
I looked at botocore. I think for here it should be ``endpoint_prefix`` to match up with botocore.